test/lib/ufe/testAttribute.py

```diff
@@ -417,19 +395,15 @@ def testAttributeFloat3(self):
         # Compare the initial UFE value to that directly from USD.
         self.assertVectorAlmostEqual(ufeAttr.get(), usdAttr.Get())
 
-        # check to see if the attribute edit is allowed
-        isAttrEditAllowed = mayaUsdUfe.isAttributeEditAllowed(usdAttr)
-        self.assertFalse(isAttrEditAllowed)
-        if isAttrEditAllowed:
-            # Set the attribute in UFE with some random values.
-            vec = ufe.Vector3f(random.random(), random.random(), random.random())
-            ufeAttr.set(vec)
-
-            # Then make sure that new UFE value matches what it in USD.
-            self.assertVectorAlmostEqual(ufeAttr.get(), usdAttr.Get())
-
-            self.runUndoRedo(ufeAttr,
-                             ufe.Vector3f(vec.x()+1.0, vec.y()+2.0, vec.z()+3.0))
+        # Set the attribute in UFE with some random values.
+        vec = ufe.Vector3f(random.random(), random.random(), random.random())
+        ufeAttr.set(vec)
+
+        # Then make sure that new UFE value matches what it in USD.
+        self.assertVectorAlmostEqual(ufeAttr.get(), usdAttr.Get())
+
+        self.runUndoRedo(ufeAttr,
+                         ufe.Vector3f(vec.x()+1.0, vec.y()+2.0, vec.z()+3.0))
```

Comment:
There is no more dead code and all my silly isAttrEditAllowed() checks are removed.